core/build.gradle.kts

```diff
@@ -66,6 +66,10 @@ configurations.api {
     // This is still experimental - additionally, it could only really benefit standalone
     exclude(group = "io.netty.incubator", module = "netty-incubator-transport-native-io_uring")
 }
+java {
+    sourceCompatibility = JavaVersion.VERSION_14
+    targetCompatibility = JavaVersion.VERSION_14
```

Comment:
Any reason the Java version was set to 14?

Reply:
https://github.com/GeyserMC/Geyser/pull/3505/commits/d6ed896b2d3da4dcf3b7f234c44f3bd614f5456a#diff-a6fced4b4e4a99784a2b92028b1adbc8c0ca28ff54e264b0e73727b3d4a460cfL69-L72